tutorials/cloud-iot-mender-ota/image/readme.md

```diff
@@ -0,0 +1,60 @@
+# Build a Mender Yocto project OS image for Raspberry Pi3 device
+
+These steps outline how to build a Yocto Project image for a Raspberry Pi3 device. 
+
+The [Yocto Project](https://www.yoctoproject.org/) is an open source collaboration project that helps developers create custom Linux-based systems for embedded products, regardless of the hardware architecture. 
+
+The build output includes a file that can be flashed to the device storage during initial provisioning, it has suffix ."sdimg. Additionally copies of the same image with ".img” and “.bmap” suffix are uploaded to GCS bucket.
+
+Yocto image builds generally take a while to complete. Generally instances with high CPU cores and memory along with faster disks such as SSD will help speed up the overall build process. Additionally, there are pre-built images for quick testing available in the GCS bucket which you can download and proceed directly to "Working with pre-built Mender Yocto Images" section
+
+Below are the instructions to build a custom Mender Yocto image for Raspberry Pi3 device. This image will have a number of requirements needed to communicate with IoT Core built in.
+
+Use the *"cloud api shell" environment you used earlier.*
+
+## Create a GCE instance for Mender Yocto Project OS builds:
+
+```
+gcloud beta compute instances create "mender-ota-build" --project $PROJECT --zone "us-central1-c" --machine-type "n1-standard-16" --subnet "default" --maintenance-policy "MIGRATE" --scopes "https://www.googleapis.com/auth/cloud-platform" --min-cpu-platform "Automatic" --tags "https-server" --image "ubuntu-1604-xenial-v20180405" --image-project "ubuntu-os-cloud" --boot-disk-size "150" --boot-disk-type=pd-ssd --boot-disk-device-name "mender-ota-build"
+```
+
+
+## SSH into the image and install the necessary updates required for the Yocto Project Builds
+
+```
+gcloud compute --project $PROJECT ssh --zone "us-central1-c" "mender-ota-build"
+```
+
+
+## Install the Mender Yocto custom image build including dependencies by downloading script from the below github repo and executing on the build server. 
+
+This step initially builds a custom image for Raspberry Pi device as well as mender artifact update which can be used to test the OTA feature of Mender. All images after the completion of the build are automatically uploaded into GCS bucket.
```

Comment:
General comment:
Use "the Raspberry Pi device" or "a Raspberry Pi device" instead of "Raspberry Pi device".